core/src/main/java/org/jdbi/v3/Jdbi.java

```diff
@@ -551,7 +551,7 @@ public void registerCollectorFactory(CollectorFactory collectorFactory)
         config.collectorRegistry.register(collectorFactory);
     }
 
-    public void registerExtensionFactory(ExtensionFactory extensionFactory)
+    public void registerExtensionFactory(ExtensionFactory<?> extensionFactory)
```

Comment:
Unrelated, but we should rename this to `registerExtension` to match `Handle`. Or vice-versa (but I prefer the shorter name).